app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
 
-
```

Comment:
The conditional statement should be indented one level deeper to align with the function definition at line 19.